html/manifest/related_applications.json

```
@@ -14,6 +14,8 @@
             },
             "edge": {
               "version_added": "79"
```

Should we go with false for now?

```suggestion
              "version_added": false
```